@here/harp-mapview/lib/ScreenProjector.ts

```diff
@@ -92,6 +92,37 @@ export class ScreenProjector {
         return undefined;
     }
 
+    /**
+     * Test if the area around the specified point is visible on the screen.
+     *
+     * @param {(Vector3Like)} source The centered source vector to project.
+     * @param {(Number)} width Half of the width of the area in NDC space [0..1].
+     * @param {(Number)} height Half of the height of the area in NDC space [0..1].
```

Comment:
they're in fractions of screen size, not in NDC space, that's why you need to multiply by 2 below.

Reply:
will do